app.js

```diff
@@ -0,0 +1,150 @@
+'use strict';
+
+var app = document.getElementById('app');
+var clicksRemaining = 25;
+var secondToLastPhotos = [];
+var previousPhotos = [];
+var displayedPhotos = [];
+
+var photos = [
+  new Photo('bag head', 'bag.jpg'),
+  new Photo('banana chopper', 'banana.jpg'),
+  new Photo('bathroom party', 'bathroom.jpg'),
+  new Photo('rain boots', 'boots.jpg'),
+  new Photo('breakfast', 'breakfast.jpg'),
+
+  new Photo('bubblegum', 'bubblegum.jpg'),
+  new Photo('chair', 'chair.jpg'),
+  new Photo('cthulhu', 'cthulhu.jpg'),
+  new Photo('dog-duck', 'dog-duck.jpg'),
+  new Photo('dragon', 'dragon.jpg'),
+
+  new Photo('pen', 'pen.jpg'),
+  new Photo('pet-sweep', 'pet-sweep.jpg'),
+  new Photo('scissors', 'scissors.jpg'),
+  new Photo('shark in the dark', 'shark.jpg'),
+  new Photo('tauntaun', 'tauntaun.jpg'),
+
+  new Photo('sweep', 'sweep.png'),
+  new Photo('unicorn', 'unicorn.jpg'),
+  new Photo('usb', 'usb.gif'),
+  new Photo('water-can', 'water-can.jpg'),
+  new Photo('wine-glass', 'wine-glass.jpg'),
+];
+
+
+renderPhotoChoices();
+
+// make a constructor for photo
+function Photo(name, filename){
+  this.name = name;
+  this.src = './imgs/' + filename;
+  this.clickCount = 0;
+  this.displayCount = 0;
+}
+
+
+
+function getRandomIndex(list) {
+  return Math.floor(Math.random() * list.length);
+}
+
+// get three random photos
+function generateRandomPhotos(){
+  photos = photos.concat(secondToLastPhotos);
```

Comment:
You can completely eliminate the secondToLastPhotos variable by moving this line to the end of this function since you only care about the previous set of images when you generate a new set of images.